internal/sliceutils/sliceutils.go

```diff
@@ -52,6 +52,31 @@ func StringSliceEquals(a, b []string) bool {
 	return true
 }
 
+// return true iff `a` has the same elements as `b`, ignoring order
```

Comment:
can we use `cmp` for this?
https://pkg.go.dev/github.com/google/go-cmp/cmp#example-Option-SortedSlice
generally more idiomatic than writing our own cmp functions